wodles/aws/buckets_s3/server_access.py

```diff
@@ -10,6 +10,7 @@
 from aws_bucket import INVALID_CREDENTIALS_ERROR_CODE, INVALID_CREDENTIALS_ERROR_MESSAGE
 from aws_bucket import INVALID_REQUEST_TIME_ERROR_CODE, INVALID_REQUEST_TIME_ERROR_MESSAGE
 from aws_bucket import THROTTLING_EXCEPTION_ERROR_CODE, THROTTLING_EXCEPTION_ERROR_MESSAGE
+from aws_bucket import UNKNOWN_ERROR_MESSAGE
```

Comment:
Even tho the way constants are being imported does not affect performance, since module imports are singletons and get cached, we should simplify code by importing the same file only one time. 

If we want to abstract the constant import from the classes or methods it can be done as well, but multiple imports of the same file to import the constants (error codes and messages) makes the code les readable.      

```suggestion
from aws_bucket import ( INVALID_CREDENTIALS_ERROR_CODE, ..., ...,  ... , UNKNOWN_ERROR_MESSAGE )
```

Reply:
I would rather define all constants in a file only for this purpose and then just:
```PYTHON
import aws_constants

...
if error_code == aws_constants.THROTTLING_EXCEPTION_ERROR_CODE:
  ...
```

However, this would be out of the scope of this issue, so I think we can do it in a new issue.